src/backend/distributed/citus--6.0-18--6.1-1.sql

```diff
@@ -0,0 +1,76 @@
+/* citus--6.0-18--6.1-1.sql */
+
+SET search_path = 'pg_catalog';
+
+ALTER TABLE pg_dist_colocation ADD COLUMN defaultgroup BOOLEAN;
+
+UPDATE pg_dist_colocation SET defaultgroup = TRUE;
+
+DROP INDEX pg_dist_colocation_configuration_index;
+
+CREATE INDEX pg_dist_colocation_configuration_index
+ON pg_dist_colocation USING btree(shardcount, replicationfactor, distributioncolumntype, defaultgroup);
+
+DROP FUNCTION create_distributed_table(regclass, text, citus.distribution_type);
+
+CREATE FUNCTION create_distributed_table(table_name regclass,
+										 distribution_column text,
+										 distribution_type citus.distribution_type DEFAULT 'hash',
+										 colocate_with text DEFAULT 'default')
```

Comment:
Hmm, I was trying out this, we have a problem.

In 3-argument variant, the last argument, the distribution method has a default: 'hash'.

Now, for 4-argument variant
- We can't use a default for the 4th argument because then PostgreSQL can not choose which function to use.
- If we don't provide a default value for 4th argument, PostgreSQL complains "ERROR:  input parameters after one with a default value must also have defaults" (This makes sense)
- We can drop distribution method and have another 3-argument variant with different parameters. This is a bit nasty because the type of the last argument is not fixed, and this would be confusing for the user. Also, the user may need to cast the last argument.
- We can remove default value of distribution method, but I think we don't want to do this.

@marcocitus, Do you see another option? If not, I will continue with the only 4-argument variant.